### PR TITLE
olsrd-defaults: Removed server 213.73.91.35 from the ping list for olsrd

### PR DESCRIPTION
**Pull request by pmelange**
See issue freifunk-berlin/firmware#551

Signed-off-by: Perry Melange <isprotejesvalkata@gmail.com>

### CODE REVIEW

**Comment by pmelange** [comment]
When the olsr watchdog notices that the olsr daemon no longer runs, it simply restarts the daemon.  But the smartgateway tunnel "tnl_+" is not deleted.  When the olsr daemon then restarts, it is unable to create a tunnel if the tunnel to the same destination already exists.  See the error log below.

```
Thu Feb  8 17:55:00 2018 user.notice OLSR watchdog: Process died - restarting!
Thu Feb  8 17:55:15 2018 daemon.info olsrd[2162]: Tunnel tnl_0ae6f052 added, to 10.230.240.82
Thu Feb  8 17:55:16 2018 daemon.info olsrd[2162]: Tunnel tnl_0ae6f052 removed, to -
Thu Feb  8 17:55:16 2018 daemon.info olsrd[2162]: Tunnel tnl_4d573032 added, to 77.87.48.50
Thu Feb  8 17:55:41 2018 daemon.info olsrd[2162]: Tunnel tnl_4d573032 removed, to -
Thu Feb  8 17:55:41 2018 daemon.err olsrd[2162]: Cannot add tunnel tnl_0a1f0b01 to 10.31.11.1: File exists (17)
Thu Feb  8 17:55:41 2018 daemon.err olsrd[2162]: Cannot create tunnel tnl_0a1f0b01
```
By deleting the tunnel and restarting olsr, everything works again.

**Comment by pmelange** [comment]
The DNS server 213.73.91.35 is offline. It is set in many of the profiles and needs to be removed.

https://twitter.com/clubdiscordia/status/983109445784875009?s=11

**Comment by pmelange** [comment]
@SvenRoederer I don't know how to add a migration or to bump the package-version.  It might be that I don't have permission to.  Some helpful advice would be great.

**Comment by pmelange** [comment]
The package version I figured out.  

But does it make sense to wait until issue freifunk-berlin/firmware#552 is resolved?

**Comment by pmelange** [comment]
Does anyone want to explain to me how to " add a migration"?

File: utils/freifunk-berlin-migration/uci-defaults/freifunk-berlin-01-migration.sh [hunk 1]
```diff
@@ -352,7 +368,11 @@ migrate () {
     r1_0_0_change_to_ffuplink
   fi
 
-  # overwrite version with the new version
+  if semverLT ${OLD_VERSION} "1.0.1"; then
```
Comment (SvenRoederer) [inline utils/freifunk-berlin-migration/uci-defaults/freifunk-berlin-01-migration.sh]: as v1.0.1 is out, this needs to go into v1.0.2-migration

**Comment by SvenRoederer** [comment]
can you squash the commits a bit?

**Comment by pmelange** [comment]
This PR has been replaced by https://github.com/freifunk-berlin/firmware-packages/pull/145

**Pull request closed without merging**